src/tennisSimulator/Menu.java

```diff
@@ -0,0 +1,68 @@
+package tennisSimulator;
+
+import java.util.Scanner;
+
+public class Menu {
+
+    public static PlayersDatabase playersDatabase = new PlayersDatabase();
+    public static Scanner scanner = new Scanner(System.in);
```

Comment:
Same

Reply:
Should these fields be `final` as Intellij suggests?